tests/Wallabag/CoreBundle/Controller/EntryControllerTest.php

```diff
@@ -1093,5 +1093,17 @@ public function testSearch()
         $crawler = $client->submit($form, $data);
 
         $this->assertCount(0, $crawler->filter('div[class=entry]'));
+
+        // test domain on list of all articles
+        $crawler = $client->request('GET', '/all/list');
+
+        $form = $crawler->filter('form[name=search]')->form();
+        $data = [
+            'search_entry[term]' => 'domain', // a string not available in the database
```

Comment:
Yes, I will update the comment to say that the filtering is not limited to domains but is applied to the whole url.

For the value used for the test, I took the value from `testFilterOnDomainName`.